### PR TITLE
[RC] Overall fix on all course pages

### PR DESCRIPTION
**Pull request by jjasferreira**
- Fix typos and punctuation;
- Improve wording and cohesion;
- Change some text style;
- Correct verb conjugations and gender of pronouns/noun determiners;
- Add a few tidbits.

### CODE REVIEW

File: content/rc/0003-transporte.md [hunk 1]
```diff
@@ -105,37 +105,37 @@ Agora considera-se que o canal pode trocar bits num segmento.
 Para os identificar, usa-se o _checksum_, já apresentado no protocolo UDP.  
 Para os resolver,
 
-- o receptor avisa o emissor que o pacote recebido estava ok, enviando um pacote
+- o recetor avisa o emissor que o pacote foi todo ele bem recebido, enviando um pacote
   ACK, ou que o pacote tinha erros, enviando um pacote NAK;
-- no caso do pacote não estar ok (ter recebido NAK), o emissor reenvia o pacote.
+- no caso do pacote não ter sido bem recebidO, o emissor reenvia o pacote, após ter recebido o NAK.
```
Comment (diogotcorreia) [inline content/rc/0003-transporte.md:110]:
```suggestion
- no caso do pacote não ter sido bem recebido, o emissor reenvia o pacote, após ter recebido o NAK.
```

File: content/rc/0003-transporte.md [hunk 2]
```diff
@@ -462,45 +462,44 @@ serra:
...
 
 ##### Variações
 
-Existem variações de implementação do AIMD. Por exemplo, em Linux é usado **TCP CUBIC** onde, se
+Existem variações de implementação do AIMD. Por exemplo, em Linux, é usado **TCP CUBIC** onde se
 assume que, quando é detetada uma congestão, esta não vai ser muito alterada.  
-Desta forma, pode-se aproximar mais rapidamente da velocidade que provocou a congestão anterior
-(visto que se assume que esta se manteve) e quando mais perto, aproximar-se mais lentamente.
+Desta forma, podemo-nos aproximar mais rapidamente da velocidade que provocou a congestão anterior
+(visto que se assume que esta se manteve) e quando mais perto estivermos, mais lentamente nos aproximamos.
```
Comment (diogotcorreia) [inline content/rc/0003-transporte.md:480]:
```suggestion
(visto que se assume que esta se manteve) e quanto mais perto estivermos, mais lentamente nos aproximamos.
```

File: content/rc/0003-transporte.md [hunk 3]
```diff
@@ -462,45 +462,44 @@ serra:
...
+que estas (que representam uma ligação maior) fiquem com uma grande parte do canal.
 
 Não existe forma de controlar isso no TCP.
 
 ### QUIC - Quick UDP Internet Connections
 
-Como mencionado anteriormente [na camada de aplicação](./0002-aplicacao.md#http-30), recentemente
-foi criado o protocolo **QUIC - Quick UDP Internet Connections**, inventada pela Google.
+Como mencionado anteriormente [na camada de Aplicação](./0002-aplicacao.md#http-30), recentemente
+foi criado o protocolo **QUIC** (_Quick UDP Internet Connections_), inventada pela Google.
```
Comment (diogotcorreia) [inline content/rc/0003-transporte.md:502]:
```suggestion
foi criado o protocolo **QUIC** (_Quick UDP Internet Connections_), inventado pela Google.
```

File: content/rc/0004-rede.md [hunk 4]
```diff
@@ -139,24 +139,24 @@ Visto que o interessante de uma máscara são o número de bits ligados (`n`), e
 ser representada ao lado de um endereço IP (`a.b.c.d`) da seguinte forma:
 
 ```
-a.b.c.d/n
+a.b.c.d /n
```
Comment (diogotcorreia) [inline content/rc/0004-rede.md:142]:
```suggestion
a.b.c.d/n
```

I don't think I've ever seen CIDR notation with spaces :thinking: 
Neither `ip -c a` nor [Wikipedia](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing) have spaces in the CIDR representation of IP addresses.

File: content/rc/0004-rede.md [hunk 5]
```diff
@@ -139,24 +139,24 @@ Visto que o interessante de uma máscara são o número de bits ligados (`n`), e
...
 ```
 
 Para o exemplo apresentado anteriormente, a máscara de rede `255.255.255.0` tem 24 bits ligados,
-por isso para o endereço IP `123.221.222.68`, sabe-se que os 24 primeiros bits representam a rede
-e os 8 bits restantes representam o _host_, podendo tudo ser representado da forma `123.221.222.68/ 24`.
+por isso, para o endereço IP `123.221.222.68`, sabe-se que os 24 primeiros bits representam a rede
+e os 8 bits restantes representam o _host_, podendo tudo ser representado da forma `123.221.222.68 /24`.
```
Comment (diogotcorreia) [inline content/rc/0004-rede.md:147]:
```suggestion
e os 8 bits restantes representam o _host_, podendo tudo ser representado da forma `123.221.222.68/24`.
```

Same here

File: content/rc/0005-ligacao-dados.md [hunk 6]
```diff
@@ -210,14 +209,15 @@ Existem vários protocolos que implementam o acesso de formas diferentes:
...
+- Dividir o canal em "pedaços" menores (intervalos de tempo, frequências, códigos, etc.);
 - Alocar um pedaço para o nó para uso exclusivo.
 
 **Alocação dinâmica ("um de cada vez")**:
 
 - Os nós tomam a vez (mas os nós com mais dados para enviar podem transmitir com mais frequência);
 - _Poll_ / seleção;
-- Passagem de um token.
+- Passagem de um _token_
+  .
```
Comment (diogotcorreia) [inline content/rc/0005-ligacao-dados.md:220]:
```suggestion
- Passagem de um _token_.
```

Probably an accidental line break

**Review by diogotcorreia — CHANGES_REQUESTED** [review]
Everything else looks great, thank you a lot!

I'll the comments unresolved in case you want to discuss some of them, otherwise resolve them and I'll merge the PR :)

**Comment by jjasferreira** [comment]
> Everything else looks great, thank you a lot!
> 
> I'll the comments unresolved in case you want to discuss some of them, otherwise resolve them and I'll merge the PR :)

Thank you for reviewing this!
You're absolutely right about the CIDR notation. 😃

**Review by diogotcorreia — APPROVED** [review]

**Pull request merged**